python_modules/libraries/dagster-databricks/dagster_databricks/configs.py

```diff
@@ -138,6 +138,83 @@ def _define_s3_storage_info():
     )
 
 
+def _define_aws_attributes_conf():
+    return Field(
+        Shape(
```

Comment:
this looks great, only recommendation is that it might be better to wrap this in Permissive rather than Shape, to make it possible to support new aws attributes without modifying this code

Reply:
Ha, that's actually how I had it originally, but thought we'd prefer to protect against misconfigurations and make any errors as explicit as possible. I'll change this back to `Permissive()`.